client/state/selectors/get-site-computed-attributes.js

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
Hi @aduth, You are totally right it is not needed outside getSite. I created separate selectors to not make getSite very huge and to make things easier to test. If you prefer I can move this part of the code inside getSite.